eth2deposit/settings.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict, NamedTuple
+
+
+class BaseChainSetting(NamedTuple):
+    GENESIS_FORK_VERSION: bytes
+
+
+MainnetSetting = BaseChainSetting(
+    GENESIS_FORK_VERSION=bytes.fromhex('00000000'),
```

Comment:
This approach is for preventing users from typoing the fork version: the users have to assign the to testnet chain *name* instead of typing the fork version *bytes* on the cli.

The downside is that we have to update this setting file for the future testnets.